apps/freelook/freelook.yml

```diff
@@ -0,0 +1,16 @@
+name: Freelook
+description: 'Freelook, an Electron-based client for Microsoft Outlook.'
```

Comment:
You can make CI happy by not starting the description with the appname :smiley: 

```text

  14105 passing (3s)
  1 failing
  1) human-submitted app data
       freelook
         freelook.yml
           should not start description with app name:
      AssertionError: expected 0 to not equal 0
      + expected - actual
```

Reply:
> You can make CI happy by not starting the description with the appname 

It's been fixed and all green :)